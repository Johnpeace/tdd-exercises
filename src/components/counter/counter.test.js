import React from 'react';
import { shallow } from 'enzyme';
import Counter from './counter.component';

describe('counter', () => {
  it('expect to render counter class component', () => {
    expect(shallow(<Counter />)).toMatchSnapshot();
  })
})
