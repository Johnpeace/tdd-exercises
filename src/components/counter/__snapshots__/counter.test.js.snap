// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`counter expect to render counter class component 1`] = `ShallowWrapper {}`;
